testing/cloudbuild-templates/cloud-run-anthos-template.cloudbuild.yaml

```diff
@@ -0,0 +1,82 @@
+steps:
+
+- id: 'Build Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['build', '-t', 'gcr.io/${PROJECT_ID}/${_SERVICE}:${COMMIT_SHA}', '$_SAMPLE']
+
+- id: 'Push Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['push', 'gcr.io/${PROJECT_ID}/${_SERVICE}:${COMMIT_SHA}']
+
+- id: 'Deploy to Cloud Run'
+  name: 'gcr.io/cloud-builders/gcloud:$_CLOUDSDK_VERSION'
+  args:
+  - 'run'
+  - 'deploy'
+  - '${_SERVICE}-${COMMIT_SHA}'
+  - '--image'
+  - 'gcr.io/${PROJECT_ID}/${_SERVICE}:${COMMIT_SHA}'
+  - '--cluster'
+  - '${_CLUSTER}'
+  - '--cluster-location'
+  - '${_SERVICE_REGION}'
+  - '--platform'
+  - 'gke'
+
+- id: 'Get Cloud Run URL'
+  name: 'gcr.io/cloud-builders/gcloud:$_CLOUDSDK_VERSION'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |
+    get_url() {
+        gcloud run services describe ${_SERVICE}-${COMMIT_SHA} --format 'value(status.url)' \
+          --platform gke --cluster ${_CLUSTER} --cluster-location ${_SERVICE_REGION}
+    }
+    echo $(get_url) > _service_url
+    echo "Cloud Run URL for ${_SERVICE} is $(cat _service_url)"
```

Comment:
Is this being used? I don't see it later in the script.

Reply:
I've updated the README to mention why we have these steps, because I don't use them later on.